src/object/srv_obj.c

```diff
@@ -2256,11 +2256,16 @@ ds_obj_tgt_update_handler(crt_rpc_t *rpc)
 	 * record/akey/dkey on some non-leader.
 	 */
 	if (DAOS_FAIL_CHECK(DAOS_VC_LOST_DATA)) {
-		if (orw->orw_dti_cos.ca_count > 0)
-			vos_dtx_commit(ioc.ioc_vos_coh,
-				       orw->orw_dti_cos.ca_arrays,
-				       orw->orw_dti_cos.ca_count, NULL);
-
+		if (orw->orw_dti_cos.ca_count > 0) {
+			rc = vos_dtx_commit(ioc.ioc_vos_coh,
+					    orw->orw_dti_cos.ca_arrays,
+					    orw->orw_dti_cos.ca_count, NULL);
+			if (rc < orw->orw_dti_cos.ca_count) {
+				D_ERROR(DF_UOID": Failed to DTX CoS commit "DF_RC".\n",
+					DP_UOID(orw->orw_oid), DP_RC(rc));
```

Comment:
Is rc always going to be an error message if this fails?  If it's a postive value, you will get rc = [value] Unknown error or some such.    Perhaps we should make this a conditional print?

Reply:
I agree. I am going to update it. Thank you!

Reply:
@jolivier23 Shall I ignore all hardware tests after I add conditional print?